rotkehlchen/db/history_events.py

```diff
@@ -148,36 +149,35 @@ def add_history_events(
                 event=event,
             )
 
-    def edit_history_event(self, event: HistoryBaseEntry) -> tuple[bool, str]:
+    def edit_history_event(self, write_cursor: 'DBCursor', event: HistoryBaseEntry) -> None:
```

Comment:
this method seems to be used only in the rest_api.py file and there the error is handled now.

The reason to raise the exception is so we can revert when editint multiple events

Reply:
this comment is addressed I guess?